src/libs/Notification/PushNotification/subscribePushNotification/index.ts

```diff
@@ -14,10 +119,14 @@ Onyx.connect({
     callback: (notificationID) => {
         if (notificationID) {
             PushNotification.register(notificationID);
-
-            // Prevent issue where report linking fails after users switch accounts without closing the app
             PushNotification.init();
-            subscribeToReportCommentPushNotifications();
+
+            // Subscribe handlers for different push notification types
+            PushNotification.onReceived(PushNotification.TYPE.REPORT_COMMENT, applyOnyxData);
+            PushNotification.onSelected(PushNotification.TYPE.REPORT_COMMENT, navigateToReport);
+
+            PushNotification.onReceived(PushNotification.TYPE.MONEY_REQUEST, applyOnyxData);
+            PushNotification.onSelected(PushNotification.TYPE.MONEY_REQUEST, navigateToReport);
```

Comment:
Do you forsee other types? I assume the individual splits will be fine, but what about the parent split action for example

Reply:
Yeah there will be other types in the future, but for now there is only `reportComment` and `moneyRequest`.